src/features/game/expansion/components/IslandList.tsx

```diff
@@ -92,25 +123,69 @@ export const IslandList = ({ bumpkin }: { bumpkin: Bumpkin | undefined }) => {
       path: `/retreat/${id}`,
     },
     {
-      name: "Farm",
+      name: "Home",
       image: human,
       levelRequired: 0,
       path: `/land/${id}`,
     },
   ];
 
+  const handleBackToHomeScreen = () => send("RETURN");
+
   const islandList = islands
     .filter((item) => item.path !== location.pathname)
     .sort((a, b) => (a.levelRequired > b.levelRequired ? 1 : -1));
 
+  // Someone who is visiting without a loaded session
+  const unAuthenticatedVisit = authState.matches("visiting");
```

Comment:
Can you be authenticated and also visiting?

Reply:
Yes you can but this state `authState.matches("visiting")` would not represent this. This is the `visiting` state on the `authMachine`. If you enter this state it means you've come via a url or you've clicked the `Explore a farm` button on the splash page. You have never loaded a session.